cluster-autoscaler/cloudprovider/gce/mig_info_provider_test.go

```diff
@@ -107,11 +109,284 @@ func (client *mockAutoscalingGceClient) CreateInstances(_ GceRef, _ string, _ in
 	return nil
 }
 
+func TestGetMigForInstanceProvider(t *testing.T) {
```

Comment:
nit: 'Provider' makes it look like there is some 'InstanceProvider' being tested.

Reply:
Done. I had to make a function name longer because of a conflict with `TestGetMigForInstance` from `autoscaling_gce_client_test.go`.